themes/bootstrap5/js/date_range_slider.js

```diff
@@ -0,0 +1,118 @@
+/*global VuFind, noUiSlider*/
+VuFind.register("dateRangeSlider", function dateRangeSlider() {
+  const _updatedSliderEvent = new Event('updated-slider');
+
+  let _defaultOptions = {
+    connect: [false, true, false],
+    step: 1,
+    format: {
+      to: (value) => Math.round(value),
+      from: (value) => Number(value)
+    },
+    handleAttributes: [
+      { 'aria-hidden': 'true', 'tabindex': '-1' },
+      { 'aria-hidden': 'true', 'tabindex': '-1' },
+    ]
+  };
+
+  /**
+   * Parse the value of an input to an integer
+   * @param {Element} input Input element
+   * @returns {number} Value of input as integer
+   */
+  function _parseInput(input) {
+    return parseInt(input.value, 10);
+  }
+
+  /**
+   * Create a slider
+   * @param {Element} sliderElement Slider base element
+   * @param {object} extraOptions Additional options for the slider. Range and starting values have to be included.
+   * @param {?array} inputs Optional array that contains two input elements that should be connected to the slider.
+   * @returns {object} Slider object
+   */
+  function create(sliderElement, extraOptions, inputs = null) {
+    let options = Object.assign({}, _defaultOptions, extraOptions);
+    let slider = noUiSlider.create(sliderElement, options);
+
+    if (inputs === null) {
+      return slider;
+    }
+
+    let inputMin = inputs[0];
+    let inputMax = inputs[1];
```

Comment:
If we keep the inputs array and have validated its length, I think we might be able to simplify this to:

```suggestion
    let [inputMin, inputMax] = inputs;
```

...though I apologize if I'm getting ahead of myself and using destructuring syntax that isn't supported by ES6.